src/components/todo-item-card/TodoItemCard.tsx

```diff
@@ -53,19 +53,23 @@ const Card = ({
       {error && <ErrorMessage error={error} />}
 
       <CardItem className={isImportant ? "border-red-600 bg-red-200" : ""}>
-        <CardItem.Body className={card.isCompleted ? "bg-gray-100" : ""}>
+        <CardItem.Body
+          className={todoItemCard.isCompleted ? "bg-gray-100" : ""}
+        >
           <CardItem.Title className="mb-4">
-            {card.title}{" "}
+            {todoItemCard.title}{" "}
```

Comment:
I believe that instead of `{" "}` you can use `margin-left` for the next `span`. 
<img width="389" alt="image" src="https://github.com/nosensus/Todo-Frontend/assets/1190436/2a25eb65-52ef-46c6-a004-13f6106a97ca">

Reply:
LOL